README.md

```diff
@@ -1,54 +1,43 @@
 # Cookiecutter Data Science
 
-_A logical, reasonably standardized, but flexible project structure for doing and sharing data science work._
+_A logical, reasonably standardized but flexible project structure for doing and sharing data science work._
 
+**Cookiecutter Data Science (CCDS)** is a tool for setting up a data science project template that incorporates best practices. To learn more about CCDS's philosophy, visit the project homepage](https://drivendata.github.io/cookiecutter-data-science/).
 
-#### [Project homepage](http://drivendata.github.io/cookiecutter-data-science/)
+> ℹ️ Cookiecutter Data Science v2 has changed from v1. It now requires installing the new cookiecutter-data-science Python package, which extends the functionality of the [cookiecutter](https://cookiecutter.readthedocs.io/en/stable/README.html) templating utility. Use the provided `ccds` command-line program instead of `cookiecutter`.
 
+## Installation
 
-### Requirements to use the cookiecutter template:
------------
- - Python 3.7+
- - [Cookiecutter Python package](http://cookiecutter.readthedocs.org/en/latest/installation.html) >= 1.4.0: This can be installed with pip by or conda depending on how you manage your Python packages:
+Cookiecutter Data Science v2 requires Python 3.7+. Since this is a cross-project utility application, we recommend installing it with [pipx](https://pypa.github.io/pipx/). Installation command options:
 
-``` bash
-$ pip install cookiecutter
-```
+```bash
+# With pipx from PyPI (recommended)
+pipx install cookiecutter-data-science
 
-or
+# With pip from PyPI
+pip install cookiecutter-data-science
 
-``` bash
-$ conda config --add channels conda-forge
-$ conda install cookiecutter
+# With conda from conda-forge (coming soon)
+# conda install cookiecutter-data-science -c conda-forge
 ```
 
+## Starting a new project
 
-### To start a new project, run:
-------------
-
-    cookiecutter -c v1 https://github.com/drivendata/cookiecutter-data-science
+To start a new project, run:
 
+```bash
+ccds https://github.com/drivendata/cookiecutter-data-science
```

Comment:
Idea to track: I guess that if we're the entrypoint now we might as well make this the default so people can just type `ccds` for the defaults

Reply:
Yeah, I was thinking that too. Will open a separate issue.